test-app-sitebricks.yaml

```diff
@@ -63,54 +63,56 @@ paths:
           description: "user was deleted successfully"
         400:
           description: "invalid user id"
-  /accounts/{userId}/balance:
+  /accounts/{userId}/tokenBalance:
     get:
-      summary: "returns user account balance"
-      operationId: "getUserBalance"
+      summary: "returns user account tokenBalance"
+      operationId: getUserBalance
       parameters:
-      - name: "userId"
+      - name: userId
         in: path
-        description: "id of user that want to get balance"
+        description: "id of user that want to get tokenBalance"
         required: true
         type: "integer"
       responses:
         200:
-          description: "return balance of user successfully"
+          description: "return tokenBalance of user successfully"
         400:
           description: "invalid user id"
-  /accounts/{userId}/deposit/{value}:
+  /accounts/{userId}/deposit:
     post:
       summary: "deposit amount to user account"
-      operationId: "deposit"
+      operationId: deposit
       parameters:
-        - name: "userId"
-          in: path
-          description: "id of user in that account we want to deposit"
+        - in: body
+          name: amount
+          description: "amount to deposit"
           required: true
-          type: "integer"
-        - name: "value"
+          schema:
+            type: "integer"
+        - name: userId
           in: path
-          description: "value that we want to deposit"
+          description: "id of user in that account we want to deposit"
           required: true
           type: "integer"
```

Comment:
kind a think you are mistaken, in order to specify that it is long, you have to add format in the next row
```
type: integer
format: int64
```